### DISCUSSION TITLE
Null vs Not Set

### DISCUSSION THREAD

**Discussion by TonyValenti** [abpframework/abp#7947] [General]
When updating an entity in ABP, we'd like to only need to pass in the fields that we're updating.

What is the correct way to differentiate between a property that has been passed in as null and one that has not been set?

**Comment by maliming** [comment]
What is your Database Provider? EF Core or MongoDB?

**Reply by TonyValenti** [reply]
EFCore.

**Reply by maliming** [reply]
You can try to use EF Core API.

https://docs.abp.io/en/abp/latest/Entity-Framework-Core#access-to-the-ef-core-api
